### PR TITLE
Dev v1.11.0

### PR DESCRIPTION
**Pull request by gschmidt-usgs**
Peer review for the dev_v1.11.0 release

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -105,22 +105,12 @@ be needed for your application or other espa product formatter libraries may nee
...
-    in the case that the fill value is not set.  This was a bug which was found
-    when converting the land/water mask to GeoTIFF.
-  * Added per-pixel angle libraries for Landsat 4-7 to support generation of
-    the per-pixel angle bands for the Collection products, which have the angle
-    coefficient files delivered with the product.
-  * Created a tool to generate the per-pixel angle bands for band 4 as the
-    representative band to be used for TOA and SR corrections.
+  * Updated the Landsat 8 angle libraries to utilize the latest version of the
+    Landsat tool as published by the Landsat team.
+  * Updated the angle band tools to support a sepcific L8 tool to generate the
```
Comment (raydittmeier) [inline README.md]: typo: "specific"

File: raw_binary/per_pixel_angles_libs/l8_ias_lib/ias_misc_write_envi_header.c [hunk 2]
```diff
@@ -185,22 +282,14 @@ int ias_misc_write_envi_header
         double corner_x = upper_left_x - (projection_distance_x / 2.0);
         double corner_y = upper_left_y + (projection_distance_y / 2.0);
 
-        /* Only supporting UTM and PS map projections at this time */
+        /* Only supporting UTM, PS, and GEO map projections at this time */
```
Comment (raydittmeier) [inline raw_binary/per_pixel_angles_libs/l8_ias_lib/ias_misc_write_envi_header.c]: The comment doesn't mention Albers being supported, but it looks like an Albers section was added.

File: README.md (hunk 1 above)
Comment (gschmidt-usgs) [inline README.md]: Fixed

File: raw_binary/per_pixel_angles_libs/l8_ias_lib/ias_misc_write_envi_header.c (hunk 2 above)
Comment (gschmidt-usgs) [inline raw_binary/per_pixel_angles_libs/l8_ias_lib/ias_misc_write_envi_header.c]: Fixed

**Pull request merged**